input/system/crunchy_bridge/system.go

```diff
@@ -0,0 +1,62 @@
+package crunchy_bridge
+
+import (
+	"time"
+
+	"github.com/pganalyze/collector/config"
+	"github.com/pganalyze/collector/input/system/selfhosted"
+	"github.com/pganalyze/collector/state"
+	"github.com/pganalyze/collector/util"
+)
+
+// GetSystemState - Gets system information about a Crunchy Bridge instance
+func GetSystemState(config config.ServerConfig, logger *util.Logger) (system state.SystemState) {
+	// With Crunchy Bridge, we are assuming that the collector is deployed on the Container Apps
+	// Most of the metrics can be obtained using the same way as the selfhosted
+	// (as the collector runs on the database server), except disk usage metrics
```

Comment:
```suggestion
	// With Crunchy Bridge, we are assuming that the collector is deployed on Container Apps,
	// which run directly on the database server. Most of the metrics can be obtained
	// using the same way as a self hosted server, since the container receives a bind mount
	// of /proc and /sys from the host. Note this excludes disk usage metrics, which we instead
	// get from the API.
```